src/scenes/Event/HistoryTable/ResultHistory/index.js

```diff
@@ -114,12 +118,22 @@ export default class EventResultHistory extends Component {
     return (
       <div>
         {resultSetsHistory.length ? (
-          <InfiniteScroll
-            spacing={0}
-            data={cards}
-            loadMore={loadMoreResultHistory}
-            loadingMore={loadingMore}
-          />
+          <Fragment>
+            <InfiniteScroll
```

Comment:
do you need InfiniteScroll if you are only going to show 5?

Reply:
Just taking advantage of the style and structure of infinitescroll, is that ok?

Reply:
ya thats fine for now. later should have a non-inifinite scroll type container for this purpose.